src/sssom/util.py

```diff
@@ -1107,13 +1107,12 @@ def get_prefix_from_curie(curie: str) -> str:
         return ""
 
 
-def curie_from_uri(uri: str, prefix_map: Mapping[str, str]) -> str:
+def curie_from_uri(uri: str, prefix_map: Union[Mapping[str, str], Converter]) -> str:
```

Comment:
Add deprecation annotation?

Reply:
I wasn't aware of this, thanks for the suggestion. Done in 118017d.